app/types/IndexPageProps.ts

```diff
@@ -14,5 +14,5 @@ export type IndexPageProps = {
   allDhbsVaccineDoseData: DhbVaccineDoseDataForIndexPage[];
   dataValidAsAtTimeUtc: string;
   dataUpdatedAtTimeUtc: string;
-  lastCheckedAtTimeUtc?: string;
+  lastCheckedAtTimeUtc: string | null;
```

Comment:
This has to be `null` - because it's serialized, so it can't be `undefined`. Decided to be explicit here, fixes it!